tokenizer/tokenizer.go

```diff
@@ -52,6 +52,14 @@ func NewWithDic(d Dic) (t Tokenizer) {
 	return Tokenizer{dic: d.dic}
 }
 
+func NewWithDicPath(p string) (Tokenizer, error) {
```

Comment:
__[Golint]__

exported function NewWithDicPath should have comment or be unexported


<!-- Comment by [Sider](https://sider.review) -->